tests/PerformanceTests/PerformanceTestService/PerformanceTestService.csproj

```diff
@@ -24,7 +24,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.ApplicationInsights.AspNetCore" Version="$(MicrosoftApplicationInsightsAspNetCoreVersion)" />
-    <PackageReference Include="Microsoft.ApplicationInsights.EventCounterCollector" Version="$(MicrosoftApplicationInsightsEventCounterCollectionVersion)" />
```

Comment:
Why replace this instead of updating the value of this variable?

Reply:
good catch. I let VS do.

Reply:
done